movement/io/save_poses.py

```diff
@@ -87,11 +186,32 @@ def to_dlc_file(ds: xr.Dataset, file_path: Union[str, Path]) -> None:
     file_path : pathlib Path or str
         Path to the file to save the DLC poses to. The file extension
         must be either ".h5" (recommended) or ".csv".
+    split_individuals : bool, optional
+        Format of the DeepLabcut output file.
```

Comment:
I would remove this line. Also, I discovered that "-" dashes do not rendr as bullet points in Sphinx (so they won't appear correctly in our API docs). It's ok to put the whole description of "split_individuals" in a single paragraph.